leda/components/AutoComplete/types.ts

```diff
@@ -96,69 +101,67 @@ export interface BlurEvent extends React.FocusEvent<HTMLInputElement> {
 }
 
 export interface AutoCompleteProps<T extends Suggestion = Suggestion> extends ValidationProps {
-  /** Браузерное автозаполнение поля ввода, по умолчанию "off" */
+  /** Browser autofill, off is the default value. Works as HTML autoComplete attribute` */
   autoComplete?: string,
-  /** Сравнение объектов по произвольному полю, а не по ссылке */
+  /** ... */
   compareObjectsBy?: T extends object ? ((suggestionListItem: T) => any) | string : never,
-  /** Данные для отображения в выпадающем списке */
+  /** Data for the dropdown list */
   data: T[],
-  /** Отключить ввод в инпуте компонента  */
+  /** In case you want to disable component  */
   isDisabled?: boolean,
-  /** Фильтр данных по правилу. smart (дефолтное значение) "умный" поиск, startsWith - фильтр по началу строку в инпуте, includes - поиск по вхождению. Не желательно использовать "умный" поиск при больших обьемах данных(1-2 тысячи значений) */
+  /** Search mode, smart is default, looks for one or several words regardless of their order, can be slow if data has thousands of elements or more */
   filterRule?: 'smart' | 'startsWith' | 'includes',
-  /** Футер под значениями в выпадающем списке */
-  footerRender?: () => React.ReactNode,
-  /** Ключ для группировки */
+  /** How to group suggestions */
   groupBy?: (option: T) => string | undefined,
-  /** Хедер над значениями в выпадающем списке */
-  headerRender?: () => React.ReactNode,
-  /** Отображение кнопки для очистки значения в инпуте */
+  /** Whether or not to show a clear button inside the input element. Default is false */
   hasClearButton?: boolean,
-  /** Вместо выпадающего списка в момент загрузки будет отображаться лоадер - полезно при подгрузке данных для списка с сервера */
+  /** Display a loading icon inside the dropdown */
   isLoading?: boolean,
-  /** Рендерить компонент с открытым выпадающим списком */
+  /** Control the dropdown state */
   isOpen?: boolean,
-  /** Кастомизация внешнего вида элемента выпадающего списка. */
+  /** Siggestion item customizator */
   itemRender?: SuggestionListProps['itemRender'],
-  /** Кастомизация внешнего вида выпадающего списка. */
+  /** Input field customizator */
   inputRender?: CustomRender<AutoCompleteProps, AutoCompleteState, InputElementProps>,
-  /** Обязательное поле или нет */
+  /** Required or not */
   isRequired?: boolean,
-  /** Кастомизация внешнего вида выпадающего списка. */
+  /** Dropdown list customizator */
   listRender?: CustomRender<SuggestionListProps, {}, UlProps>,
-  /** Количество символов в инпуте после которых начать показывать выпадающий список */
+  /** The minimal number of symbols that triggers the dropdown opening */
   minSearchLength?: number,
-  /** Имя поля ввода */
+  /** Component name */
   name?: string,
   /** This will be shown when no suggestions are found */
   noSuggestionsText?: React.ReactNode,
-  /** Принимает JSX */
+  /** No suggetions block customizator */
```

Comment:
Block who? Maybe a different term code be used
```suggestion
  /** No suggetions label customizator */
```